src/chttpd_cors.erl

```diff
@@ -306,6 +316,35 @@ get_cors_config(#httpd{cors_config = undefined}) ->
 get_cors_config(#httpd{cors_config = Config}) ->
     Config.
 
+cors_config(Host, Key, Default) ->
+    config:get(cors_section(Host), Key,
+                     config:get("cors", Key, Default)).
+
+cors_section(Host0) ->
+    {Host, _Port} = split_host_port(Host0),
```

Comment:
Just a note: `couch_httpd_vhost:host/1` returns value of either `X-Forward-Host` (as setup by apache's mod_proxy, for example) or `Host`. Neither include scheme, not when setup by browser, at least, so `HostAsString` is actually plain host:port string. I imagine CORS config setting as noted in default.ini right now just doesn't work, the working config expects [cors:Host] format.